clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -880,10 +880,14 @@ public void handle(OffsetCommitResponse commitResponse, RequestFuture<Void> futu
                             log.error("Received fatal exception: group.instance.id gets fenced");
                             future.raise(error);
                             return;
+                        } else if (error == Errors.REBALANCE_IN_PROGRESS) {
+                            // need to re-join group but do not need to reset generation
```

Comment:
Do we need this comment?

Reply:
Yeah, the comment is not adding much. Perhaps if you explain why we do not need to reset generation? Otherwise, we can just remove it.